### DISCUSSION TITLE
Unwanted weapons crates in Sandbox and Missions (Custom Gamemodes)

### DISCUSSION THREAD

**Discussion by AziKarnage** [FakeFishGames/Barotrauma#15347] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When loading up a station or map there are often crates full of guns usually light arms like pistols or smgs, which are totally abused or unwanted. This issue we have not been able to toggle on or off, and when it occurs it disrupts balance and causes more chaos than what security or admins want/can handle.

### Reproduction steps

Start a public server like a station rp
Go to storage area where supplies would be deposited
Find typical crates as well as a crate full of guns

mostly stations have reported the issue and it may be more predominantly sandbox or mission mod, but Custom Gamemodes use both

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Other

### -

Both dedicated and independent servers

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I'm not entirely sure if this is an issue in the game or a mod issue. To my knowledge, there should be no crates with weapons in the outposts in the vanilla game modes (aside from outpost deathmatch). Could it be the crates have been pre-placed in the custom outpost, or that there's some kind of custom logic that spawns them there? Which mod(s) does this happen in specifically?
